pyproject.toml

```diff
@@ -13,14 +13,15 @@ requires-python = ">= 3.10"
 dependencies = [
     "ctakesclient >= 3, < 4",
     "delta-spark >= 2.3, < 3",
+    "fhirclient < 5",
     "httpx < 1",
     "inscriptis < 3",
     "jwcrypto < 2",
     "label-studio-sdk < 1",
     "oracledb < 2",
     "pandas < 3",
     "philter-lite < 1",
-    "pyarrow < 12",
+    "pyarrow < 13",
```

Comment:
Not necessary, I just noticed we were a version back while I was playing with data transforms.